===================================================================
RCS file: /repository/php-src/scripts/phpize.in,v
retrieving revision 1.1.2.7
retrieving revision 1.1.2.9
diff -p --unified=3 -r1.1.2.7 -r1.1.2.9
--- scripts/phpize.in	2003/08/27 17:57:32	1.1.2.7
+++ scripts/phpize.in	2003/11/14 03:21:15	1.1.2.9
@@ -10,7 +10,7 @@ FILES="acinclude.m4 Makefile.global"
 CLEAN_FILES="$FILES *.lo *.la *.o .deps .libs/ build/ include/ modules/ install-sh \
     mkinstalldirs missing config.nice config.sub config.guess configure configure.in \
 	aclocal.m4 config.h config.h.in conftest* ltmain.sh libtool config.cache \
-	config.log config.status Makefile Makefile.fragments Makefile.objects"
+	config.log config.status Makefile Makefile.fragments Makefile.objects confdefs.h"

 case "$1" in
   # Cleanup
@@ -57,6 +57,13 @@ touch install-sh mkinstalldirs missing
 aclocal || exit 1
 autoconf || exit 1
 autoheader || exit 1
+test -x $builddir/build/shtool || chmod +x $builddir/build/shtool
+if test ! -x $builddir/build/shtool; then
+  echo "shtool at '$builddir/build/shtool' not executable. "
+  echo "Make sure that the file exists and is executable and then rerun this script. "
+  echo
+  exit 1
+fi
 libtoolize=`$builddir/build/shtool path glibtoolize libtoolize`
 $libtoolize -f -c || exit 1

